### ISSUE TITLE
Latest image complains about missing sudo and exits right away

### ISSUE THREAD

**Issue by jackxuhong** [AshDevFr/docker-spigot#19]
The latest image doesn't seem to work, please advise, thanks.
```
$ docker run -it -p 25565:25565 ashdev/minecraft-spigot:latest
/spigot_init.sh: line 6: sudo: command not found
```

**Comment by Nykaer** [comment]
Just tested on ubuntu and a Centos hostOS.  sudo present on both systems, and path seems ok.  

Still throws 'sudo not found' error

**Comment by jackxuhong** [comment]
Nykaer, sudo is missing in the docker image, not the host system.

**Comment by Nykaer** [comment]
Yup, and I believe  'RUN apt-get install -y software-properties-common **sudo**' should fix that. But it doesn't.

**Comment by QuietKerb** [comment]
Thanks Nykaer,

You're right my fix was not working, I had actually used apt to test which does work.
I've updated my pull request to use apt instead as apt-get was getting 404s on some repos when installing sudo dependencies by the looks of it.

This is the change you can make: 8e582a845dbf5dcb1e4ca73d1b70fa1f0fee20fa
